admin-commands/create-muterole.md

```diff
@@ -15,13 +15,9 @@ None
 ### **Usage**
 
 ```
-m:create-muterole
+/create-muterole
```

Comment:
Behavior of this command changed (no interactive prompt, takes an optional arg which is the role to set mute permissions, if none then creates a new role)

Reply:
Updated